### PR TITLE
Docker build

### PR DESCRIPTION
**Pull request by helinwang**
Fixes: https://github.com/PaddlePaddle/Paddle/issues/1655

### CODE REVIEW

File: paddle/scripts/docker/build.sh [hunk 1]
```diff
@@ -1,144 +1,110 @@
...
     pip install --upgrade pip && \
-    pip install -U 'protobuf==3.1.0' requests
-RUN pip install numpy
+    pip install -U 'protobuf==3.1.0' requests numpy
 # Use different deb file when building different type of images
-ADD \$PWD/${DEB_PATH}*.deb /usr/local/opt/paddle/deb/
-RUN dpkg --force-all -i /usr/local/opt/paddle/deb/*.deb && rm -f /usr/local/opt/paddle/deb/*.deb
+ADD build/*.deb /usr/local/opt/paddle/deb/
+ADD build/py_paddle*linux*.whl /usr/local/opt/paddle/share/wheels/
+ADD build/paddle*.whl /usr/local/opt/paddle/share/wheels/
```
Comment (typhoonzero) [inline paddle/scripts/docker/build.sh]: 这三个ADD没必要哈，因为deb文件中已经包含了paddle python的库了。下面执行paddle version会自动安装

File: paddle/scripts/docker/build.sh [hunk 2]
```diff
@@ -1,144 +1,110 @@
...
+
+# run paddle version to install python packages first
+RUN dpkg -i /usr/local/opt/paddle/deb/*.deb && rm -f /usr/local/opt/paddle/deb/*.deb && paddle version
 
 ENV PATH="/usr/local/opt/paddle/bin/:${PATH}"
```
Comment (typhoonzero) [inline paddle/scripts/docker/build.sh]: 这个ENV也可以去掉了paddle 会安装在/usr/bin下

File: paddle/scripts/docker/build.sh (hunk 1 above)
Comment (helinwang) [inline paddle/scripts/docker/build.sh]: Done. Removed last two, I think the first one is necessary.

File: paddle/scripts/docker/build.sh (hunk 2 above)
Comment (helinwang) [inline paddle/scripts/docker/build.sh]: Done.

**Pull request merged**